test/functional/page_objects/visual_builder_page.ts

```diff
@@ -727,4 +727,18 @@ export class VisualBuilderPageObject extends FtrService {
       await this.comboBox.set('topHitOrderByFieldSelect', timeField);
     });
   }
+
+  public async setFilterRatioNumerator(numerator: string) {
+    const numeratorInput = await this.find.byCssSelector(
+      '.tvbAggRow [data-test-subj="queryInput"]'
+    );
+    await numeratorInput.type(numerator);
+  }
+
+  public async setFilterRatioDenominator(denominator: string) {
+    const [, denominatorInput] = await this.find.allByCssSelector(
+      '.tvbAggRow [data-test-subj="queryInput"]'
+    );
+    await denominatorInput.type(denominator);
```

Comment:
can we create private function with common logic and in these methods just return needed input?